packages/client/components/ActivityLibrary/ActivityLibrary.tsx

```diff
@@ -62,20 +101,57 @@ export const ActivityLibrary = (props: Props) => {
   const {
     query: searchQuery,
     filteredItems: filteredTemplates,
-    onQueryChange
+    onQueryChange,
+    resetQuery
   } = useSearchFilter(templates, getTemplateValue)
 
+  const [selectedCategory, setSelectedCategory] = useState<CategoryID>('recommended')
+
+  const templatesToRender = useMemo(() => {
+    if (searchQuery.length > 0) {
+      // If there's a search query, just use the search filter results
+      return filteredTemplates
+    }
+
+    return filteredTemplates.filter((template) =>
+      selectedCategory === 'recommended'
+        ? template.isRecommended
+        : template.category === selectedCategory
+    )
+  }, [searchQuery, filteredTemplates, selectedCategory])
+
   if (!featureFlags.retrosInDisguise) {
     return <Redirect to='/404' />
   }
 
+  const onSelectCategory = (category: CategoryID) => {
+    setSelectedCategory(category)
+    resetQuery()
+  }
+
   return (
     <div className='flex h-full bg-white'>
       <ActivityLibrarySideBar />
       <div>
         <SearchBar searchQuery={searchQuery} onChange={onQueryChange} />
+        <div className='ml-2 flex'>
+          {(Object.keys(CATEGORY_ID_TO_NAME) as Array<CategoryID>).map((category) => (
+            <button
```

Comment:
+1 we probably want each tab to be it's own URL so we can link to it from landing pages etc. If that's the case, then it'd be better to use `a` instead of `button` here and use `nav` as a parent to be a good internet citizen.

@mattkrick any thoughts?